src/libcore/alloc.rs

```diff
@@ -213,18 +213,19 @@ impl Layout {
     /// Creates a layout by rounding the size of this layout up to a multiple
     /// of the layout's alignment.
     ///
-    /// Returns `Err` if the padded size would overflow.
-    ///
     /// This is equivalent to adding the result of `padding_needed_for`
     /// to the layout's current size.
     #[unstable(feature = "alloc_layout_extra", issue = "55724")]
     #[inline]
-    pub fn pad_to_align(&self) -> Result<Layout, LayoutErr> {
+    pub fn pad_to_align(&self) -> Layout {
         let pad = self.padding_needed_for(self.align());
-        let new_size = self.size().checked_add(pad)
-            .ok_or(LayoutErr { private: () })?;
+        // This cannot overflow: it is an invariant of Layout that
+        // > `size`, when rounded up to the nearest multiple of `align`,
```

Comment:
What are these `>` saying here?

Reply:
It's supposed to be a markdown quote from `from_size_align`'s docs.

Reply:
It looks like a glitch.^^

Maybe edit the intro then, to something like
```
// This cannot overflow. Quoting from the invariant of Layout:
```

Reply:
Done.